### ISSUE TITLE
How to use both TabController and AnimationController in the same widget in Flutter.

### ISSUE THREAD

**Issue by MrNinja** [flutter/flutter#44636]
## Steps to Reproduce

1. Create a new `StatefulWidget` called `ListPage`.
2. Create a `TabBarView` inside `_ListPageState` which have TickerProviderStateMixin.
3. Create a `LinearProgressIndicator` inside `_ListPageState`.

=> This code below makes my screen freeze. None of `TabBarView` are loaded. `LinearProgressIndicator` do not show.

    _animationController = AnimationController(
            duration: const Duration(milliseconds: 2000), vsync: this);

**Target Platform**: Android
**Target OS version/browser**: Android 8.0.0
**Devices**: Samsung Galaxy Note 8

Today I got a problem when I tried to create a `LinearProgressIndicator` side by side a `TabBarView` in Flutter. I searched Flutter docs and some example on Github but fails.

**TL;DR**:

Today I got a problem when I tried to create a `LinearProgressIndicator` side by side a `TabBarView`. My source got no problem if I create the new customs `ListWidget extends StatefulWidget` with only `TabBarView`. However, this code below makes my screen freeze. None of `TabBarView` are loaded. `LinearProgressIndicator` do not show.

    _animationController = AnimationController(
            duration: const Duration(milliseconds: 2000), vsync: this);

**My source code**: (I deleted some unrelated code)

    class ListPage extends StatefulWidget {
      @override
      _ListPageState createState() => _ListPageState();
    }
    
    class _ListPageState extends State<ListPage> with TickerProviderStateMixin {
      TabController _tabController;
      AnimationController _animationController;
      Animation<double> _animation;
      List<Tab> tabs;
    
      @override
      void initState() {
        super.initState();
        TabsRepository tabsRepository = TabsRepository();
        tabs = tabsRepository.getTabs();
        _tabController = TabController(length: tabs.length, vsync: this);
        _animationController = AnimationController(
            duration: const Duration(milliseconds: 2000), vsync: this);
        _animation = Tween(begin: 0.0, end: 1.0).animate(_animationController);
        _animationController.repeat();
      }
    
      @override
      void dispose() {
        _tabController.dispose();
        _animationController.stop();
        _animationController.dispose();
        super.dispose();
      }
    
      @override
      Widget build(BuildContext context) {
        return Scaffold(
          body: Container(
            child: Column(
              children: <Widget>[
                LinearProgressIndicator(value: _animation.value),
                TabBarView(
                  controller: _tabController,
                  children: tabs.map((tab) => CardsTabView(tab.text)).toList(),
                ),
              ],
            ),
          ),
        );
      }
    }

**My `flutter doctor` message**:

Windows PowerShell

    Copyright (C) Microsoft Corporation. All rights reserved.
    
    PS C:\Users\rikimaru\source\flutter\project> flutter doctor
    Doctor summary (to see all details, run flutter doctor -v):
    [√] Flutter (Channel stable, v1.9.1+hotfix.6, on Microsoft Windows [Version 10.0.17134.1069], locale ja-JP)
    
    [√] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    [√] Android Studio (version 3.5)
    [√] VS Code (version 1.40.0)
    [√] Connected device (1 available)
    
    • No issues found!

**I tried some solution**:

 - I read the docs from `api.flutter.dev`.
 - I use dummy data instead of data from api.
 - I find the different between `SingleTickerProviderStateMixin` and `TickerProviderStateMixin`. However, [this project][1] on Github worked for me.
 - I rebuild and reinstall my app instead of hot reload.

  [1]: https://github.com/flutter-prj/Animation-CatInTheBox/blob/master/lib/src/screens/home.dart

**Comment by MrNinja** [comment]
### Logs
**`flutter run --verbose`**:

```
[   +5 ms] Built build\app\outputs\apk\debug\app-debug.apk.
[   +5 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\build-tools\29.0.2\aapt dump xmltree
C:\Users\rikimaru\...\build\app\outputs\apk\app.apk AndroidManifest.xml
[  +55 ms] Exit code 0 from: C:\Users\rikimaru\AppData\Local\Android\sdk\build-tools\29.0.2\aapt dump xmltree
C:\Users\rikimaru\...\build\app\outputs\apk\app.apk AndroidManifest.xml
[   +1 ms] N: android=http://schemas.android.com/apk/res/android
             E: manifest (line=2)
               A: android:versionCode(0x0101021b)=(type 0x10)0x1
               A: android:versionName(0x0101021c)="1.0.0" (Raw: "1.0.0")
               A: android:compileSdkVersion(0x01010572)=(type 0x10)0x1c
               A: android:compileSdkVersionCodename(0x01010573)="9" (Raw: "9")
               A: package="com.example.project" (Raw: "com.example.project")
               A: platformBuildVersionCode=(type 0x10)0x1
               A: platformBuildVersionName="1.0.0" (Raw: "1.0.0")
               E: uses-sdk (line=7)
                 A: android:minSdkVersion(0x0101020c)=(type 0x10)0x10
                 A: android:targetSdkVersion(0x01010270)=(type 0x10)0x1c
               E: uses-permission (line=14)
                 A: android:name(0x01010003)="android.permission.INTERNET" (Raw: "android.permission.INTERNET")
               E: application (line=22)
                 A: android:label(0x01010001)="project" (Raw: "project")
                 A: android:icon(0x01010002)=@0x7f020000
                 A: android:name(0x01010003)="io.flutter.app.FlutterApplication" (Raw: "io.flutter.app.FlutterApplication")
                 A: android:debuggable(0x0101000f)=(type 0x12)0xffffffff
                 E: activity (line=27)
                   A: android:theme(0x01010000)=@0x7f030000
                   A: android:name(0x01010003)="com.example.project.MainActivity" (Raw: "com.example.project.MainActivity")
                   A: android:launchMode(0x0101001d)=(type 0x10)0x1
                   A: android:configChanges(0x0101001f)=(type 0x11)0x400037b4
                   A: android:windowSoftInputMode(0x0101022b)=(type 0x11)0x10
                   A: android:hardwareAccelerated(0x010102d3)=(type 0x12)0xffffffff
                   E: meta-data (line=41)
                     A: android:name(0x01010003)="io.flutter.app.android.SplashScreenUntilFirstFrame" (Raw: "io.flutter.app.android.SplashScreenUntilFirstFrame")
                     A: android:value(0x01010024)=(type 0x12)0xffffffff
                   E: intent-filter (line=45)
                     E: action (line=46)
                       A: android:name(0x01010003)="android.intent.action.MAIN" (Raw: "android.intent.action.MAIN")
                     E: category (line=48)
                       A: android:name(0x01010003)="android.intent.category.LAUNCHER" (Raw: "android.intent.category.LAUNCHER")
[   +1 ms] Stopping app 'app.apk' on SC 01K.
[   +1 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e shell am force-stop
com.example.project
[ +257 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e shell pm list packages
com.example.project
[ +687 ms] package:com.example.project
[   +3 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e shell cat
/data/local/tmp/sky.com.example.project.sha1
[ +108 ms] 88b4b5c3cbea688cc536af57d768c7d6a233c82d
[   +1 ms] Installing APK.
[   +2 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe version
[  +53 ms] Android Debug Bridge version 1.0.41
           Version 29.0.5-5949299
           Installed as C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe
[   +1 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe start-server
[  +51 ms] Installing build\app\outputs\apk\app.apk...
[   +1 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e install -t -r
C:\Users\rikimaru\...\build\app\outputs\apk\app.apk
[+17947 ms] Performing Streamed Install
                     Success
[   +1 ms] Installing build\app\outputs\apk\app.apk... (completed in 17.9s)
[   +3 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e shell echo -n
9321f3cc5a20210f2730add3606f858d3a52ac24 > /data/local/tmp/sky.com.example.project.sha1
[ +100 ms] SC 01K startApp
[   +2 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e shell am start -a android.intent.action.RUN -f
0x20000000 --ez enable-background-compilation true --ez enable-dart-profiling true --ez enable-checked-mode true --ez verify-entry-points true
com.example.project/com.example.project.MainActivity
[ +202 ms] Starting: Intent { act=android.intent.action.RUN flg=0x20000000 cmp=com.example.project/.MainActivity (has extras) }
[   +1 ms] Waiting for observatory port to be available...
[+1001 ms] Observatory URL on device: http://127.0.0.1:35115/cglABlvk3z0=/
[   +6 ms] executing: C:\Users\rikimaru\AppData\Local\Android\sdk\platform-tools\adb.exe -s ce061716a17a4926017e forward tcp:0 tcp:35115
[  +68 ms] 64842
[        ] Forwarded host port 64842 to device port 35115 for Observatory
[   +7 ms] Connecting to service protocol: http://127.0.0.1:64842/cglABlvk3z0=/
[ +352 ms] Successfully connected to service protocol: http://127.0.0.1:64842/cglABlvk3z0=/
[   +3 ms] Sending to VM service: getVM({})
[  +12 ms] Result: {type: VM, name: vm, architectureBits: 64, hostCPU: Qualcomm Technologies, Inc MSM8998, operatingSystem: android, targetCPU: arm64, version: 2.5.0(Fri Sep 6 20:10:36 2019 +0200) on "android_arm64", _profilerMode: VM, _nativeZoneMemoryUsage: 0, pi...
[   +9 ms] Sending to VM service: getIsolate({isolateId: isolates/3979506474631999})
[   +6 ms] Sending to VM service: _flutter.listViews({})
[  +10 ms] Result: {type: FlutterViewList, views: [{type: FlutterView, id: _flutterView/0x7bb1e54420, isolate: {type: @Isolate, fixedId: true, id:
isolates/3979506474631999, name: main.dart$main-3979506474631999, number: 3979506474631999}}]}
[  +26 ms] DevFS: Creating new filesystem on the device (null)
[   +2 ms] Sending to VM service: _createDevFS({fsName: project})
[  +48 ms] Result: {type: Isolate, id: isolates/3979506474631999, name: main, number: 3979506474631999, _originNumber: 3979506474631999, startTime: 1573527062550,
_heaps: {new: {type: HeapSpace, name: new, vmName: Scavenger, collections: 0, avgCollectionPeriodMillis...
[  +63 ms] Result: {type: FileSystem, name: project, uri:
file:///data/user/0/com.example.project/code_cache/projectRVFVBY/project/}
[  +19 ms] DevFS: Created new filesystem on the device
(file:///data/user/0/com.example.project/code_cache/projectRVFVBY/project/)
[   +4 ms] Updating assets
[ +184 ms] Scanned through 0 files in 0ms
[   +2 ms] Syncing files to device SC 01K...
[   +2 ms] Scanning asset files
[   +4 ms] <- reset
[        ] Compiling dart to kernel with 0 updated files
[   +8 ms] C:\flutter-sdk\bin\cache\dart-sdk\bin\dart C:\flutter-sdk\bin\cache\artifacts\engine\windows-x64\frontend_server.dart.snapshot --sdk-root
C:\flutter-sdk\bin\cache\artifacts\engine\common\flutter_patched_sdk/ --incremental --strong --target=flutter --output-dill build\app.dill --packages
C:\Users\rikimaru\...\.packages --filesystem-scheme org-dartlang-root
[  +12 ms] <- compile package:project/main.dart
[ +158 ms] -> result 7899d90e-a0f7-4326-9ea8-a16c37ed88d7
[ +187 ms] D/libGLESv2(18960): STS_GLApi : DTS, ODTC are not allowed for Package : com.example.project
[ +207 ms] I/flutter (18960): ══╡ EXCEPTION CAUGHT BY RENDERING LIBRARY ╞═════════════════════════════════════════════════════════
[   +2 ms] I/flutter (18960): The following assertion was thrown during performResize():
[        ] I/flutter (18960): Horizontal viewport was given unbounded height.
[   +1 ms] I/flutter (18960): Viewports expand in the cross axis to fill their container and constrain their children to match
[        ] I/flutter (18960): their extent in the cross axis. In this case, a horizontal viewport was given an unlimited amount of
[        ] I/flutter (18960): vertical space in which to expand.
[        ] I/flutter (18960): 
[        ] I/flutter (18960): Widget creation tracking is currently disabled. Enabling it enables improved error messages. It can
[        ] I/flutter (18960): be enabled by passing `--track-widget-creation` to `flutter run` or `flutter test`.
[        ] I/flutter (18960): 
[        ] I/flutter (18960): When the exception was thrown, this was the stack:
[        ] I/flutter (18960): #0      RenderViewport.performResize.<anonymous closure> (package:flutter/src/rendering/viewport.dart:1206:15)
[   +1 ms] I/flutter (18960): #1      RenderViewport.performResize (package:flutter/src/rendering/viewport.dart:1218:6)
[        ] I/flutter (18960): #2      RenderObject.layout (package:flutter/src/rendering/object.dart:1686:9)
[        ] I/flutter (18960): #3      RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #4      RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #5      RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #6      RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #7      RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #8      RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #9      RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #10     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #11     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #12     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #13     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #14     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #15     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #16     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #17     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #18     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #19     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +2 ms] I/flutter (18960): #20     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +4 ms] I/flutter (18960): #21     RenderFlex.performLayout (package:flutter/src/rendering/flex.dart:744:15)
[   +1 ms] I/flutter (18960): #22     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #23     MultiChildLayoutDelegate.layoutChild (package:flutter/src/rendering/custom_layout.dart:142:11)
[        ] I/flutter (18960): #24     _ScaffoldLayout.performLayout (package:flutter/src/material/scaffold.dart:444:7)
[        ] I/flutter (18960): #25     MultiChildLayoutDelegate._callPerformLayout (package:flutter/src/rendering/custom_layout.dart:212:7)
[   +1 ms] I/flutter (18960): #26     RenderCustomMultiChildLayoutBox.performLayout (package:flutter/src/rendering/custom_layout.dart:356:14)
[   +1 ms] I/flutter (18960): #27     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #28     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #29     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #30     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #31     _RenderCustomClip.performLayout (package:flutter/src/rendering/proxy_box.dart:1232:11)
[   +1 ms] I/flutter (18960): #32     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #33     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #34     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #35     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #36     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #37     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #38     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +2 ms] I/flutter (18960): #39     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #40     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #41     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #42     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #43     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +6 ms] I/flutter (18960): #44     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +2 ms] I/flutter (18960): #45     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #46     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #47     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #48     RenderOffstage.performLayout (package:flutter/src/rendering/proxy_box.dart:3140:13)
[        ] I/flutter (18960): #49     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #50     RenderStack.performLayout (package:flutter/src/rendering/stack.dart:510:15)
[        ] I/flutter (18960): #51     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +2 ms] I/flutter (18960): #52     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #53     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #54     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #55     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #56     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +2 ms] I/flutter (18960): #57     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +1 ms] I/flutter (18960): #58     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #59     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #60     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[        ] I/flutter (18960): #61     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[        ] I/flutter (18960): #62     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:105:13)
[   +1 ms] I/flutter (18960): #63     RenderObject.layout (package:flutter/src/rendering/object.dart:1701:7)
[   +2 ms] I/flutter (18960): #64     RenderView.performLayout (package:flutter/src/rendering/view.dart:152:13)
[   +2 ms] I/flutter (18960): #65     RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1578:7)
[   +1 ms] I/flutter (18960): #66     PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:844:18)
[   +5 ms] I/flutter (18960): #67     RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:341:19)
[   +2 ms] I/flutter (18960): #68     WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:761:13)
[  +20 ms] I/flutter (18960): #69     RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:280:5)
[   +9 ms] I/flutter (18960): #70     SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1033:15)
[   +9 ms] I/flutter (18960): #71     SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:975:9)
[        ] I/flutter (18960): #72     SchedulerBinding.scheduleWarmUpFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:784:7)
[        ] I/flutter (18960): #81     _Timer._runTimers (dart:isolate-patch/timer_impl.dart:382:19)
[   +1 ms] I/flutter (18960): #82     _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:416:5)
[   +1 ms] I/flutter (18960): #83     _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:172:12)
[        ] I/flutter (18960): (elided 8 frames from package dart:async and package dart:async-patch)
[        ] I/flutter (18960): 
[   +2 ms] I/flutter (18960): The following RenderObject was being processed when the exception was fired: RenderViewport#a8371 NEEDS-LAYOUT NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE:
[   +2 ms] I/flutter (18960):   needs compositing
[        ] I/flutter (18960):   creator: Viewport ← IgnorePointer-[GlobalKey#40b6e] ← Semantics ← _PointerListener ← Listener ←
[  +35 ms] I/flutter (18960):     _GestureSemantics ← RawGestureDetector-[LabeledGlobalKey<RawGestureDetectorState>#42bd6] ←
[   +3 ms] I/flutter (18960):     _PointerListener ← Listener ← _ScrollableScope ← _ScrollSemantics-[GlobalKey#47d13] ←
[   +1 ms] I/flutter (18960):     RepaintBoundary ← ⋯
[  +10 ms] I/flutter (18960):   parentData: <none> (can use size)
[   +2 ms] I/flutter (18960):   constraints: BoxConstraints(0.0<=w<=411.4, 0.0<=h<=Infinity)
[   +1 ms] I/flutter (18960):   size: MISSING
[   +1 ms] I/flutter (18960):   axisDirection: right
[   +1 ms] I/flutter (18960):   crossAxisDirection: down
[   +5 ms] I/flutter (18960):   offset: _PagePosition#1e5db(offset: null, range: null..null, viewport: null, ScrollableState,
[   +1 ms] I/flutter (18960):     PageScrollPhysics -> PageScrollPhysics -> ClampingScrollPhysics -> ClampingScrollPhysics,
[   +2 ms] I/flutter (18960):     IdleScrollActivity#f9609, ScrollDirection.idle)
[   +2 ms] I/flutter (18960):   anchor: 0.0
[   +1 ms] I/flutter (18960): This RenderObject had the following child:
[   +1 ms] I/flutter (18960):     center child: RenderSliverFillViewport#f4dcd NEEDS-LAYOUT NEEDS-PAINT
[        ] I/flutter (18960): ════════════════════════════════════════════════════════════════════════════════════════════════════
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderViewport#a8371 NEEDS-LAYOUT NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
[   +2 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderViewport#a8371 NEEDS-PAINT NEEDS-COMPOSITING-BITS-UPDATE
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderIgnorePointer#7e1c0 relayoutBoundary=up10 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderSemanticsAnnotations#6033e relayoutBoundary=up9 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +2 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderPointerListener#52629 relayoutBoundary=up8 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +3 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderSemanticsGestureHandler#92c7d relayoutBoundary=up7 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderPointerListener#e1ab9 relayoutBoundary=up6 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: _RenderScrollSemantics#710a4 relayoutBoundary=up5 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderRepaintBoundary#36ec1 relayoutBoundary=up4 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[  +18 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderCustomPaint#ce45d relayoutBoundary=up3 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +5 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderRepaintBoundary#caf83 relayoutBoundary=up2 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +1 ms] I/flutter (18960): Another exception was thrown: RenderBox was not laid out: RenderFlex#cd0b8 relayoutBoundary=up1 NEEDS-PAINT
NEEDS-COMPOSITING-BITS-UPDATE
[   +6 ms] I/flutter (18960): Another exception was thrown: NoSuchMethodError: The method '>' was called on null.
[ +777 ms] I/zygote64(18960): Do partial code cache collection, code=30KB, data=24KB
[   +3 ms] I/zygote64(18960): After code cache collection, code=30KB, data=24KB
[   +8 ms] I/zygote64(18960): Increasing code cache capacity to 128KB
[+1516 ms] -> 7899d90e-a0f7-4326-9ea8-a16c37ed88d7
[        ] -> +file:///C:/flutter-sdk/.pub-cache/hosted/pub.dartlang.org/sprintf-4.0.2/lib/src/sprintf_impl.dart
... Dummy message I think.
[        ] -> +
[        ] -> 7899d90e-a0f7-4326-9ea8-a16c37ed88d7 build\app.dill 0
[        ] Updating files
[ +442 ms] DevFS: Sync finished
[   +1 ms] Syncing files to device SC 01K... (completed in 4,221ms, longer than expected)
[   +1 ms] Synced 4.2MB.
[   +1 ms] Sending to VM service: _flutter.listViews({})
[   +9 ms] Result: {type: FlutterViewList, views: [{type: FlutterView, id: _flutterView/0x7bb1e54420, isolate: {type: @Isolate, fixedId: true, id:
isolates/3979506474631999, name: main.dart$main-3979506474631999, number: 3979506474631999}}]}
[   +3 ms] <- accept
[   +1 ms] Connected to _flutterView/0x7bb1e54420.
[   +6 ms] 🔥  To hot reload changes while running, press "r". To hot restart (and rebuild state), press "R".
```

**Comment by MrNinja** [comment]
Hi, development team!
I'm apologize for the inconvenience. That's my bad.
I read the `flutter run --verbose` output carefully again and got the reason of this problems after that. I have to wrap my `TabBarView` inside `Expanded`.
